functional/subscription_order_test.py

```diff
@@ -0,0 +1,63 @@
+import os
+import unittest
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+
+class SubscriptionOrderTests(unittest.TestCase):
+    def setUp(self):
+        self.BASE_URL = os.getenv("BASE_URL", "")
+        self.driver = webdriver.Chrome()
+
+    def tearDown(self):
+        self.driver.close()
+
+    def test_subscription_order(self):
+        self.driver.get(os.path.join(self.BASE_URL, "subscribe"))
+
+        assert self.driver.title == "Order Subscription - Paint Drying"
+
+        self.driver.find_element(by=By.ID, value="email").send_keys("a@b.c")        # write email address to email field
+        self.driver.find_element(by=By.ID, value="subscription_level-0").click()    # check bronze subscription
+        self.driver.find_element(by=By.ID, value="submit").click()                  # submit form
+
+        assert self.driver.title == "Paint Drying"
+        assert self.driver.current_url == self.BASE_URL                             # check if redirected to main
+
+    def test_subscription_order_return_to_main(self):
+        self.driver.get(os.path.join(self.BASE_URL, "subscribe"))
+
+        assert self.driver.title == "Order Subscription - Paint Drying"
```

Comment:
Same thing as in main_test.py file. `title` was already asserted in test_subscription_order().